WordPress/WordPressTest/BlogBuilder.swift

```diff
@@ -37,9 +37,9 @@ final class BlogBuilder {
 
     func withJetpack(version: String? = nil, username: String? = nil, email: String? = nil) -> Self {
         set(blogOption: "jetpack_client_id", value: 1)
-        set(blogOption: "jetpack_version", value: version)
-        set(blogOption: "jetpack_user_login", value: username)
-        set(blogOption: "jetpack_user_email", value: email)
+        set(blogOption: "jetpack_version", value: version as Any)
+        set(blogOption: "jetpack_user_login", value: username as Any)
+        set(blogOption: "jetpack_user_email", value: email as Any)
```

Comment:
@emilylaguna I noticed these typecast while looking over https://github.com/wordpress-mobile/WordPress-iOS/pull/16883.

I'm curious to know what made them necessary?

Reply:
Yeah, Xcode 12.5 was complaining about them. 

<img width="970" alt="Screen Shot 2021-07-20 at 9 09 40 AM" src="https://user-images.githubusercontent.com/793774/126329708-210d129e-a925-4c6c-98a5-6632900ac6bb.png">